dom.bs

```diff
@@ -816,7 +818,14 @@ must return the values they were initialized to.
 The
 <dfn method for=Event>preventDefault()</dfn>
 method must set the <a>canceled flag</a> if the
-{{Event/cancelable}} attribute value is true.
+{{Event/cancelable}} attribute value is true and
+the <a>in passive listener flag</a> is unset.
+
+<p class="note no-backref">
+ User agents are encouraged to generate a console warning or other debugging
+ aid to help authors identify places where calls to {{preventDefault()}}
+ have no effect.
```

Comment:
Ok, how's [this](https://github.com/RByers/dom/commit/e3a2df3c7df30d68247d758c7f746ed76375b92c#diff-a467e681501e456ce8c1ef31425e1b41R825)?